documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -3280,7 +3280,9 @@ If you select this option, to prevent the connector from emitting and processing
 |The setting determines the value for link:https://www.postgresql.org/docs/current/sql-altertable.html#SQL-ALTERTABLE-REPLICA-IDENTITY[replica identity] at table level. +
  +
 This option will overwrite the existing value in database.
-A comma-separated list of regular expressions that match fully-qualified tables and replica identity value to be used in the table. +
+A comma-separated list of regular expressions that match fully-qualified tables and replica identity value to be used in the table.
+The changes will be applied only to captured tables.
```

Comment:
Thanks for the update, this is really helpful.

Could you add at the end withcaptured tables.

Maybe you could expand with where are the captured tables are defined with:
 
The changes will be applied only to captured tables, e.g. tables have been included in the config `table.include.list`.